app/routines/update_user_salesforce_info.rb

```diff
@@ -0,0 +1,148 @@
+class UpdateUserSalesforceInfo
+  USER_BLOCK_SIZE = 1000
+
+  def initialize
+    @errors = []
+  end
+
+  def self.call
+    new.call
```

Comment:
Are we giving up on Lev? Why not use a Lev routine with `transaction: :no_transaction`?